### DISCUSSION TITLE
How to resize svg element

### DISCUSSION THREAD

**Discussion by ghost** [marc2332/freya#954] [Q&A] [answered]
Hi,

I have tried to load linux application icons and tried to display them with the `svg` element. It works fine but I'm not able to define the size for the svg element and the svg image. Width and height in the svg files is set to `128`. And if I define a size on the `svg` element the svg image is always displayed in 128*128. This is what I have tried.

```
svg {
    width: "12"
    height: "12",
    svg_data: icon_data,
}
```

Resizing works fine for the `image` element.

**Comment by marc2332** [comment, marked as answer]
Yes, this is a known limitation at the moment, rust-skia doesn't have an API that I can use to force a certain size for an SVG. I have a PR open https://github.com/marc2332/freya/pull/850 but I am still waiting for https://github.com/rust-skia/rust-skia/pull/1030 to be merged

**Reply by ghost** [reply]
Ok thank you for your response. Then I will wait until it is merged.

**Reply by ghost** [reply]
By the way. You have done a really good job with Freya. I enjoy it a lot :-)

**Reply by marc2332** [reply]
> By the way. You have done a really good job with Freya. I enjoy it a lot :-)

Thank you! I appreciate your kind words 🦀
